src/common/enum_util.cpp

```diff
@@ -2228,6 +2229,34 @@ ForeignKeyType EnumUtil::FromString<ForeignKeyType>(const char *value) {
 	throw NotImplementedException(StringUtil::Format("Enum value: '%s' not implemented", value));
 }
 
+template<>
```

Comment:
Does this actually get serialised?

Reply:
I don't think so. But running `generate_enum_util.py` adds this.